tests/rptest/services/machinetype.py

```diff
@@ -71,6 +75,13 @@ class MachineTypeConfig:
     MachineTypeName.IM4GN_8XLARGE:
     MachineTypeConfig(num_shards=31, memory=128 * GiB),
 
+    MachineTypeName.M7GD_LARGE:
+    MachineTypeConfig(num_shards=1, memory=8 * GiB),
```

Comment:
I see, so these are cloud-centric definitions which already bake in the N-1 kubernetes thing (but then use the full memory amount?). Seems a bit weird but consistent with all the existing enums.

Reply:
Good point yeah, it's all bit meh.